modulefiles/tasks/wcoss2/make_ics.local.lua

```diff
@@ -18,3 +18,5 @@ load(pathJoin("nemsio", os.getenv("nemsio_ver")))
 load(pathJoin("udunits", os.getenv("udunits_ver")))
 load(pathJoin("gsl", os.getenv("gsl_ver")))
 load(pathJoin("nco", os.getenv("nco_ver")))
+
+setenv("BLENDINGPYTHON","/apps/spack/python/3.8.6/intel/19.1.3.304/pjn2nzkjvqgmjw4hmyz43v5x4jbxjzpk/bin/python")
```

Comment:
@hu5970, what is the difference between BLENDINGPYTHON and general(?) python? Does it use any specific packages that are not installed in python on wcoss2?

Reply:
@chan-hoo, sorry if there is confusion. This is just the version of (general) python that I was using on WCOSS2 for the blending function. It does not use any other specific packages that were not already installed in python on WCOSS2.

Reply:
@delippi, I got it. Thanks. If so, @MatthewPyle-NOAA, we can merge this PR as it is. However, we'll need to remove this 'BLENDINGPYTHON' in the near future, and set up a new python environment including additional python packages such as netCDF4 and numpy on other machines (Hera, Jet, Orion) because the current "workflow_tool" env may not have them as far as I know. I'll open an issue for this.

Reply:
@chan-hoo Any objection to working on this PR before your PR to clean up the home directory (and other things from the AQM review)?

Reply:
Nope. :)

Reply:
@chan-hoo @delippi Thanks for working on python environment  setup. We do have problem to use python in different machines. There are several other tasks also using python (smoke/dust preprocess and fvcom preprocess). Need a good python environment works for those tasks on Hera/Jet/WCOSS2.